spec/models/attendance_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Attendance, type: :model do
+  context 'testing associations' do
+    it 'should have many events' do
+      a = Attendance.reflect_on_association(:event_attendee)
+      expect(a.macro).to eq(:belongs_to)
+    end
+    it 'should have many users' do
+      a = Attendance.reflect_on_association(:attended_event)
+      expect(a.macro).to eq(:belongs_to)
+    end
+  end
+end
```

Comment:
### use require_relative which is the rails generic way of loading files
![rspec](https://user-images.githubusercontent.com/8114528/84964212-354e2f00-b114-11ea-9437-211eb4ae7c61.png)